blockchain/processing.py

```diff
@@ -216,6 +220,42 @@ def get_prior_hash(self, origin_id, phase):
 
         return prior_hash
 
+    def _subscription_request(self, transaction):
+        """ check if given transaction has one or more subscriptions tied to it and inserts into subscriptions database
+            initial communication with subscription node
```

Comment:
Can you split and re-order this comment -- second line is the generic func desc and first should be _I think_ a line comment before the conditional.